[ASM] - EXPANDR -  1578 (#24380)
demisto/content: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules_README.md

> This playbook returns "RemediationAction" options based return from Remediation Path Rules API, or defaults to data collection task options from "Cortex ADM - Decision" subplaybook.

Reviewer: ```suggestion
This playbook returns "RemediationAction" options based on the return from the Remediation Path Rules API, or defaults to data collection task options from the "Cortex ADM - Decision" sub-playbook.
```

## Dependencies
This playbook uses the following sub-playbooks, integrations, and scripts.

### Sub-playbooks

> * Cortex ASM - Decision

Reviewer: ```suggestion
Cortex ASM - Decision
```

### Integrations

> * CortexAttackSurfaceManagement

Reviewer: ```suggestion
CortexAttackSurfaceManagement
```

### Scripts
* Set
* GetTime
* RemediationPathRuleEvaluation

### Commands

> * asm-list-remediation-rule

Reviewer: ```suggestion
asm-list-remediation-rule
```

## Playbook Inputs
---

| **Name** | **Description** | **Default Value** | **Required** |
| --- | --- | --- | --- |

> | ExternallyDetectedProviders | Providers of external service. |  | Optional |

Reviewer: ```suggestion
| ExternallyDetectedProviders | Providers of the external service. |  | Optional |
```
